fulcrumapp/fulcrum-schema
# story/FLCRM-8922 compareForms should recognize schema_version

**Proposed Changes:**

This PR introduces a "schema_version" on a Form/App, which can be used in place of "options.version". This allows changing/migrating system table columns.

The current use case is for the jira-like keys (ABC-123) feature, and introduces:

- `record_key` and `record_sequence` on `form_<id>` tables
- ~`record_record_key` and `record_record_sequence` on `form_<id>_<repeatablekey>` tables~

**Testing Steps:**

The unit tests were updated (thank goodness there were tests to begin with!!!), and the `dist/fulcrum-schema.min.js` was copied into a branch on Fulcrum that can provide Forms with different schema_versions.

Additionally, this code was tested by the Kraken in https://fulcrumapp.atlassian.net/browse/FLCRM-8964 to verify existing functionality is unaffected.
